membrane_curvature/data/test_po4_small.gro

```diff
@@ -9,4 +9,4 @@ Test file 9 lipids in grid
  2708POPC   PO4    7   0.000   2.000  12.000
  2713POPC   PO4    8   1.000   2.000  12.000
  2999POPC   PO4    9   2.000   2.000  12.000
-  3.00000  3.00000  3.00000   
+  3.00000  3.00000  30.00000   
```

Comment:
I'm sorry for not noticing this before. The fact that the box has `u.dimension[2]=30 A` (or 3nm), made the tests in lines 303 and 315 fail. Since 120 is a multiple of 30, the `derived_surface` was zero and therefore, everything else was zero.  Hope it makes sense.

Reply:
Ah, thanks for that explanation, I wouldn't have known where to look otherwise!